js/fieldmanager-datepicker.js

```diff
@@ -13,4 +13,8 @@
 	fmLoadModule( fm.datepicker.add_datepicker );
 
 	$( document ).on( 'fm_collapsible_toggle fm_added_element fm_displayif_toggle fm_activate_tab', fm.datepicker.add_datepicker );
+	$( document ).on( 'focus',
+			'input[class*="fm-datepicker-group"]:not(.fm-has-date-picker)',
+			fm.autocomplete.enable_autocomplete
+		);
```

Comment:
- The indentation of the last three lines seems inconsistent.
- `add_datepicker`, not `enable_autocomplete`?